test/fastdom-test.js

```diff
@@ -340,7 +340,7 @@ suite('fastdom', function() {
 
     test('it removes reference to the job if cleared', function(done) {
       var write = sinon.spy();
-      var id = fastdom.mutate(2, write);
+      var id = fastdom.mutate(function () { return false; }, write);
```

Comment:
I'm confused by this test, maybe there was a typo initially. I think it should have been:

```js
var id = fastdom.mutate(write);
```